### ISSUE TITLE
Mod, options, and save screens are blank

### ISSUE THREAD

**Issue by jzebedee** [BUTR/Bannerlord.BUTRLoader#137]
Affected version: v1.21.0
Last known good: v1.20.1

Steam version v1.0.3.9055 of Bannerlord with `en-us` locale.

No files blocked:
```powershell
PS C:\Games\Steam\steamapps\common\Mount & Blade II Bannerlord\bin\Win64_Shipping_Client> Get-Item * -Stream "Zone.Identifier" -ErrorAction SilentlyContinue
PS C:\Games\Steam\steamapps\common\Mount & Blade II Bannerlord\bin\Win64_Shipping_Client>
```

https://user-images.githubusercontent.com/1896945/211948709-9976fc42-40a7-4f51-8ce0-2ae4ae5da957.mp4

**Comment by Aragas** [comment]
Thanks for the report! v1.21.1/2 should have fixed the issue, could you confirm?

**Comment by jzebedee** [comment]
Thanks for the quick fix! As of v1.21.2 these screens are loading correctly.
